### ISSUE TITLE
Text Positioning

### ISSUE THREAD

**Issue by herwig4** [jgraph/drawio-desktop#1345]
Hi,
when I insert a TEXT into a form by double-click, there´s no possibility to set the position of the text.
Only after inserting the text. Clicking anywhere else, clicking the text again and then there is a menu, where I can set the position.
Please show the POSITIONING-set while entering a text.
That´s all.

* [ ] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio-desktop/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [ ] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by herwig4** [comment]
wontfix = completed??
I'm sorry for not being understood. 
thanks for asking no questions.
